mlflow/transformers.py

```diff
@@ -52,31 +52,36 @@
 from mlflow.utils.requirements_utils import _get_pinned_requirement
 
 FLAVOR_NAME = "transformers"
-_PIPELINE_BINARY_KEY = "pipeline"
-_PIPELINE_BINARY_FILE_NAME = "pipeline"
+
+_CARD_TEXT_FILE_NAME = "model_card.md"
+_CARD_DATA_FILE_NAME = "model_card_data.yaml"
 _COMPONENTS_BINARY_KEY = "components"
-_INFERENCE_CONFIG_BINARY_KEY = "inference_config.txt"
-_MODEL_KEY = "model"
-_TOKENIZER_KEY = "tokenizer"
 _FEATURE_EXTRACTOR_KEY = "feature_extractor"
-_IMAGE_PROCESSOR_KEY = "image_processor"
-_PROCESSOR_KEY = "processor"
-_TOKENIZER_TYPE_KEY = "tokenizer_type"
 _FEATURE_EXTRACTOR_TYPE_KEY = "feature_extractor_type"
+_FRAMEWORK_KEY = "framework"
+_IMAGE_PROCESSOR_KEY = "image_processor"
 _IMAGE_PROCESSOR_TYPE_KEY = "image_processor_type"
-_PROCESSOR_TYPE_KEY = "processor_type"
-_CARD_TEXT_FILE_NAME = "model_card.md"
-_CARD_DATA_FILE_NAME = "model_card_data.yaml"
-_TASK_KEY = "task"
+_INFERENCE_CONFIG_BINARY_KEY = "inference_config.txt"
 _INSTANCE_TYPE_KEY = "instance_type"
-_PIPELINE_MODEL_TYPE_KEY = "pipeline_model_type"
+_MODEL_KEY = "model"
 _MODEL_PATH_OR_NAME_KEY = "source_model_name"
-_SUPPORTED_SAVE_KEYS = {_MODEL_KEY, _TOKENIZER_KEY, _FEATURE_EXTRACTOR_KEY, _IMAGE_PROCESSOR_KEY}
```

Comment:
To confirm, are all of these changes backwards compatible?

Reply:
Yep, it's just alphabetical ordering of the names (it was hurting my eyes seeing them all out of order). I ran through the full local test suite (that we can't run on CI) to ensure that all tests pass and did a validation manually of all keys and values.

Reply:
I'd usually separate this type of cosmetic change.